.goreleaser-dev.yml

```diff
@@ -32,6 +32,16 @@ builds:
       - -s
       - -w
       - -X github.com/checkmarx/ast-cli/internal/params.Version={{.Version}}
+    hooks:
+      post:
+        - cmd: bash .github/scripts/signing_win.sh dist/cx_windows_amd64_v1/cx.exe {{.Os}} || true
+          output: true
```

Comment:
is this the check for dev? if you are exit with 1 it'll not fail everything?

Reply:
No because the condition 'or true' will cause this line to continue with the process without failing the pipeline